src/axom/inlet/inlet_utils.cpp

```diff
@@ -12,5 +12,21 @@ void setWarningFlag(axom::sidre::Group* root) {
   }
 }
 
+std::string getFullName(const std::string& prefix, const std::string& name)
+{
+  if (prefix == "")

```

Comment:
Can you update this to a Ternary Operator?  I admit still I'm forcing myself to get used to them.

```
std::string getFullName(const std::string& prefix, const std::string& name)
{
    return (prefix == "") ? name : prefix + "/" + name;
}
```